packages/snabela/snabela.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+maintainer: "mmatalka@gmail.com"
+build: [
+	[make "-j%{jobs}%"]
+]
+
+build-test: [
+	[make "-j%{jobs}%" "test"]
+]
+
+install: [
+	[make "PREFIX=%{prefix}%" "install"]
+]
+
+remove: [
+	[make "PREFIX=%{prefix}%" "remove"]
+]
+
+depends: [
+	"cmdliner"
+	"containers"
+	"merlin-of-pds"
+	"ocamlfind"
+	"oth" { test }
+	"pds" { build & (>= "5" & < "6") }
+	"ppx_deriving"
+	"process"
+	"sedlex"
+	"toml"
+]
+
+authors: [
+	"mmatalka@gmail.com"
+]
+
+homepage: "https://bitbucket.org/acslab/snabela"
+bug-reports: "https://bitbucket.org/acslab/snabela/issues"
+dev-repo: "git@bitbucket.org:acslab/snabela.git"
+
```

Comment:
needs to be constrained to ocaml >=4.03.0 , or the `result` compatibility package needs to be added

```
# File "../../../src/snabela/snabela_lexer.ml", line 148, characters 4-6:
# Error: Unbound constructor Ok
# make[2]: *** [snabela_lexer.cmx] Error 2
# make[1]: *** [release_snabela] Error 2
# make: *** [all] Error 2
```